source/manual/business-readiness-update-content.html.md

```diff
@@ -5,15 +5,15 @@ section: Business readiness finder
 layout: manual_layout
 parent: "/manual.html"
 important: true
-last_reviewed_on: 2019-03-26
+last_reviewed_on: 2019-04-10
 review_in: 3 months
 ---
 
 The [business readiness finder][business-readiness-finder] relies on metadata tags in search-api to filter content.
 
 ## Updating content
 
-Content is curated and added to a [spreadsheet][] by the content team. Developer help is needed to upload the content. We do this by converting the spreadsheet into a CSV, including it in [govuk-app-deplopment-secrets][govuk-app-deployment-secrets] and then releasing search-api.
+Content is curated and added to a [spreadsheet][] by the content team. Developer help is needed to upload the content. We do this by converting the spreadsheet into a CSV, including it in [govuk-app-deplopment-secrets][govuk-app-deployment-secrets] and then releasing `search-api`.
```

Comment:
Can we correct `govuk-app-deplopment-secrets` to `govuk-app-deployment-secrets`?

Reply:
Thanks @vanitabarrett - took me ages to spot the difference! 😄